### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by ales1979** [monicahq/monica#6352] [Q&A]
I am looking for help installing on my hosting

**Comment by pheonix-devapps** [comment]
Same here

**Comment by fabriziotoppi** [comment]
can it be installed on hosting? procedure?

**Comment by ciphermenial** [comment]
What type of install? A manual install, where you install it on a Linux distribution? A docker install? What are you requesting.

**Comment by itsrsy** [comment]
me too
need some guide to install it on shared hosting like godaddy 
it will be helpful to have instructions similar as developer setup but what works on shared hosting

**Reply by slifty** [reply]
I could be wrong but I'm not sure you can set this up on shared hosting -- if you're not comfortable running docker, then I suspect installing packages manually is going to actually be *more* of a burden.

At that point you might want to consider just paying the Monica team directly for their [hosted solution](https://www.monicahq.com/) .  This is a nice approach because now instead of paying GoDaddy, you're paying the people developing the software and increasing the odds of future updates and improvements!

**Reply by itsrsy** [reply]
I am already paying hosting provider and this will be free for me
If we have to pay and keep my data with them then it doesn't make any difference with open source or closed source 

Anyways I am looking if it's possible to use this software on shared hosting like GoDaddy
If not then it's another open source project free for enterprises only
For privacy oriented people it will be too costly to use source code and they will indirectly be forced to use the hosted version as you mentioned

**Reply by slifty** [reply]
Understood.  I do believe that the answer to your specific question is not without some R&D / and possibly not at all (i.e. it isn't easy to use this on shared hosting) but I could definitely be wrong!

You may want to check with your shared hosting provider to see if they have any way to run Docker containers.  If they do, then you might be in luck!  You could also look at the [docker layers](https://hub.docker.com/layers/library/monica/4.0.0/images/sha256-9e4270cde2bbb51b07dafa3f8df6ddc9f7fdb2507fc03a993deddd12f5b20737?context=explore) to see what is actually involved in setting up the project in a linux environment.

That said...

It sounds like you're looking for something that is [free as in beer](https://en.wiktionary.org/wiki/free_as_in_beer) rather than [free as in speech](https://en.wiktionary.org/wiki/free_as_in_speech#English).  This project offers both flavors of free (though, to your point, the solutions they have provided might not align with your very specific use case.)

Remember there is absolutely still consumer value in using software that is open source even if they are paying someone to host it.  The fact that you have the ability to take your data out of a hosted version and set up your own copy is, in itself, a liberating force!

Good luck on your efforts.

**Reply by itsrsy** [reply]
I partially disagree
If using benefits of free and open source is costlier then non free hosted service option then there is no sense of having that project as open source for small business or individuals

Cost of running it makes a big difference it may be a business strategy for Monica
But it's not only this project many open source doing the same to make earning

They make the project open source but make it hard for anyone to use it easily either by technical means or by making it costly to use open source

Anyways thanks

**Comment by aoirthoir** [comment]
It is written in php right? Should'nt it be just as simple as dropping it in a directory? Maybe the public directory? I dunno since I cannot find installation instructions.
